From 18d28cf9c669d7fb411035d652a92f6f399ddc5e Mon Sep 17 00:00:00 2001
From: jschyra <jschyra@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 17 Nov 2011 17:55:41 +0000
Subject: [PATCH 0200/1076]  - Nachrichten wurden im Hobbit falsch angezeigt.
 Somit werden Leerzeichen jetzt auch wieder normal als solche deklariert.

git-svn-id: svn://localhost/SelfServiceCommon/trunk@242 90b65887-3827-0410-9a23-83215b262276
---
 .../cpp/LogService/HOBBIT/bbWin/Kiosk.xslt       | 16 ++++++++--------
 .../cpp/LogService/HOBBIT/bbWin/Paper.xslt       |  8 ++++----
 2 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
index aca18a95..bf5229a3 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
+++ b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Kiosk.xslt
@@ -21,13 +21,13 @@
                 <xsl:text>red</xsl:text>
             </xsl:otherwise>
         </xsl:choose>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:value-of select="translate(@ts,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:value-of select="translate(@name,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:value-of select="translate(@st,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:text>&#x0A;</xsl:text>
         <xsl:apply-templates select="COMP" />
     </xsl:template>
@@ -87,17 +87,17 @@
         </xsl:choose>
         <xsl:text></xsl:text>
         <xsl:value-of select="translate(@name,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:value-of select="translate(@st,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:value-of select="translate(@ts,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
         <xsl:apply-templates select="REASON" />
         <xsl:text>&#x0A;</xsl:text>
     </xsl:template>
     
     <xsl:template match="REASON">
         <xsl:value-of select="translate(@name,'&#xA;',' ')"/>
-        <xsl:text>&#160;</xsl:text>
+        <xsl:text> </xsl:text>
     </xsl:template>
 </xsl:stylesheet>
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
index 3152254f..63debe53 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
+++ b/SelfServiceCommon/Massai/cpp/LogService/HOBBIT/bbWin/Paper.xslt
@@ -34,13 +34,13 @@
                     <xsl:text>red</xsl:text>
                 </xsl:otherwise>
             </xsl:choose>
-            <xsl:text>&#160;</xsl:text>
+            <xsl:text> </xsl:text>
             <xsl:value-of select="translate(@ts,'&#xA;',' ')" />
-            <xsl:text>&#160;</xsl:text>
+            <xsl:text> </xsl:text>
             <xsl:value-of select="translate(@name,'&#xA;',' ')" />
-            <xsl:text>&#160;</xsl:text>
+            <xsl:text> </xsl:text>
             <xsl:value-of select="translate(@st,'&#xA;',' ')" />
-            <xsl:text>&#160;</xsl:text>
+            <xsl:text> </xsl:text>
             <xsl:text>&#x0A;</xsl:text>
         </xsl:if>
     </xsl:template>
-- 
2.41.0.windows.1

